src/components/marketplace/buy_sell.tsx

```diff
@@ -151,9 +151,11 @@ const TokenTextButtonUppercase = styled.span`
 class BuySell extends React.Component<Props, State> {
     public state = {
         makerAmount: new BigNumber(0),
+        orderFeeEther: new BigNumber(0),
```

Comment:
Is this still used?

Reply:
@Agupane yup, it looks like the fee-handling code is not needed anymore here: attributes in the component's state and `updateOrderFee` method (since everything is encapsulated in `OrderDetails`).

Reply:
now it's removed